homeassistant/components/plex/config_flow.py

```diff
@@ -254,22 +258,55 @@ class PlexOptionsFlowHandler(config_entries.OptionsFlow):
     def __init__(self, config_entry):
         """Initialize Plex options flow."""
         self.options = copy.deepcopy(config_entry.options)
+        self.server_id = config_entry.data[CONF_SERVER_IDENTIFIER]
```

Comment:
You use this once, no need to store it?

Reply:
I needed to grab it from the `config_entry` parameter for use in `async_step_plex_mp_settings` which doesn't have access. Should I just store the `plex_server` as a local attribute from the start?

Reply:
Yeah! Also: you should use config entry unique id 👍

Reply:
I remember why I did this now. The `self.hass` object wasn't available during `__init__`, but _was_ once I entered a step.

Reply:
Config entry unique ID is a good idea, but I think this PR has enough changes as of now. I'll add it to my todo list.